Gordon360/Documentation/Gordon360.xml

```diff
@@ -560,6 +554,15 @@
             <returns>The updated news item</returns>
             <remarks>The news item must be authored by the user and must not be expired and must be unapproved</remarks>
         </member>
+        <member name="M:Gordon360.Controllers.NewsController.EditPostingImage(System.Int32,System.String)">

```

Comment:
Hmm.  Why is this added in this PR?

Reply:
The documentation file gets auto-updated based on the XML documentation for each route. If this was manually added, it should be removed. Otherwise it should be okay.